### PR TITLE
Add March 2023 Updates blog post

### PR DESCRIPTION
**Pull request by ricekot**
Also,
- Add a README and script to generate content for the blog post easily.
- Remove broken image paths for bronzeV1 sponsors.

Please let me know if I've missed anything :).

<details><summary>Screenshot</summary>

image

</details>

### CODE REVIEW

**Review by psiinon — COMMENTED** [review]
Thanks!
I think there are more website updates, Inc changing the sponsorship levels and at least one new sponsor. On my phone so harder to check 😉

File: site/data/supporters.yaml [hunk 1]
```diff
@@ -73,17 +73,14 @@ bronzeV1:
...
     notes: $3,000 in donations
 
   - name: 'HostedScan'
     link: https://hostedscan.com/
-    logo: /img/supporters/hostedscan.png
     notes: $1,500 donation
 
   - name: 'HiSolution AG'
     link: https://www.hisolutions.com/
-    logo: /img/supporters/hisolutions.png
```
Comment (kingthorin) [inline site/data/supporters.yaml:86]: Did these exist at some point and should we be re-adding them?

Comment (thc202) [reply site/data/supporters.yaml:86]: Bronze v1 do not use images, that's only for v2. (They did exist at some point.)

File: site/content/blog/2023-04-01-zap-updates-march-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+**Authentication**  
+We are actively working on improving the authentication experience in ZAP.
+For example, changes to add automatic detection and configuration of header-based session management to ZAP are being tested actively at the moment.
+
+**Selenium 4**  
+[Selenium](https://www.selenium.dev/) v4 has been out for a while and we are actively working on updating ZAP to use it.
+There is a chance that this update could break existing scripts - more details when we have them.
+
+**Form Handler RegEx Support**  
+You will soon be able to match parameters names in HTTP requests using regular expressions using the FormHandler add-on.
```
Comment (thc202) [inline site/content/blog/2023-04-01-zap-updates-march-2023/index.md:63]: `Form Handler`

File: site/content/blog/2023-04-01-zap-updates-march-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+**GraphQL IDE**  
+The GraphQL add-on is being updated to include [GraphiQL](https://github.com/graphql/graphiql/).
+The GraphQL IDE will be exposed via the ZAP API and all requests made using the IDE will be proxied through ZAP.
+
+**NodeJS API Client**
```
Comment (thc202) [inline site/content/blog/2023-04-01-zap-updates-march-2023/index.md:69]: The latest .NET APi was released in March.

File: site/content/blog/2023-04-01-zap-updates-march-2023/index.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+Added
+- A context menu active on the output text areas facilitating the replacement of input text.
+- PowerShell encoder (hat tip to hackvertor/hackvertor#71 for the idea and details).
+
+##### Browser View (v6)
+Added
+- Add info and repo URLs.
+
+Changed
+- Update minimum ZAP (vto 2.12.0.)
```
Comment (thc202) [inline site/content/blog/2023-04-01-zap-updates-march-2023/index.md:157]: Typos?

File: site/data/supporters.yaml (hunk 1 above)
Comment (psiinon) [reply site/data/supporters.yaml:86]: They got demoted in the sponsors review - which should be mentioned 😉
Re the updated webpages - another script could help...

Comment (kingthorin) [reply site/data/supporters.yaml:86]: Okay

Comment (thc202) [reply site/data/supporters.yaml:86]: We could add a zapbot monthly action that PRs the base blog posts (filling those automated parts).

File: site/content/blog/2023-04-01-zap-updates-march-2023/index.md (hunk 4 above)
Comment (ricekot) [reply site/content/blog/2023-04-01-zap-updates-march-2023/index.md:157]: Yeah, I had done a find-and-replace and this got caught by accident.

Comment (kingthorin) [reply site/content/blog/2023-04-01-zap-updates-march-2023/index.md:157]: Been there 😉

**Comment by ricekot** [comment]
Tweaked. Thanks for your comments, I think they should be addressed now.

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**